Shevchuk_Maksym/README.md

```diff
@@ -0,0 +1,93 @@
+Discord Registration and Usage Guide
+Introduction
+Discord is a popular communication platform that enables users to chat, voice call, and video call with individuals and groups. This document provides a step-by-step guide on how to register and use Discord effectively.
+
+Table of Contents
+Creating a Discord Account
+Setting up Discord
+Joining a Server
+Navigating the Discord Interface
+Sending Messages
+Voice and Video Calls
+Managing Notifications
+Customizing Discord
+Privacy and Security
+Troubleshooting
+1. Creating a Discord Account
+To start using Discord, you need to create a user account. Follow these steps:
+
+1)Visit the Discord website at discord.com using your preferred web browser.
+2)Click on the "Login" button located in the top-right corner of the page.
+3)On the login page, click on the "Register" button.
+4)Enter your email address, a secure password, and a unique username.
```

Comment:
The fields are listed in the following order: Email, Username, Password, the Date of birth. Please specify if all the fields are required. Minimum age is required to access the app or website. For most countries it's 13. It would be good to add a note about this requirement.